commcare/src/main/java/org/motechproject/commcare/service/CaseActionService.java

```diff
@@ -0,0 +1,13 @@
+package org.motechproject.commcare.service;
+
+import org.motechproject.commcare.domain.CaseTask;
+
+import java.util.Map;
+
+/**
+ * This service is responsible for handling "Create Case" and "Update Case" actions in tasks.
```

Comment:
change the class javadoc. There should not be info about methods in this service. Just general information. Imagine that in the future, there will be more actions, so you will have to update class javadoc everytime